ui/push-health/Health.jsx

```diff
@@ -31,12 +37,27 @@ export default class Health extends React.PureComponent {
       result: null,
       failureMessage: null,
       notifications: [],
+      progressExpanded: true,
+      lintingExpanded: false,
+      buildsExpanded: false,
+      testsExpanded: false,
+      performanceExpanded: false,
     };
   }
 
   componentDidMount() {
     // Get the test data
-    this.updatePushHealth();
+    this.updatePushHealth().then(data => {
```

Comment:
Should this use the `getData` wrapper instead of `then`?

Reply:
Oh shoot.  I should have just used an ``await`` on ``updatePushHealth`` because that function uses ``getData``.  Thanks for catching that.